class UserNotExistsException(Exception):
    pass


class UserExistsException(Exception):
    pass
